src/contracts/fungible/data/asset.rs

```diff
@@ -163,7 +163,7 @@ impl AddAssign for AccountingAmount {
 )]
 #[display(Debug)]
 pub struct Asset {
-    id: ContractId, // This is a unique primary key
+    pub id: ContractId, // This is a unique primary key
```

Comment:
as I said in the PR description, I wasn't sure that commit was necessary. it is there because I was trying to change IDs of each `Asset` from the vector received in the sdk. clearly this commit is not necessary since we decided to wait a rgb-node release using bech32 (as discussed in LNP-BP/rgb-sdk#31)